pkg/transforms/node.go

```diff
@@ -51,6 +51,14 @@ func NodeResourceBuilder(n *v1.Node) *NodeResource {
 	node.Properties["_systemUUID"] = strings.TrimRight(n.Status.NodeInfo.SystemUUID, "\000")
 	node.Properties["role"] = roles
 
+	conditions := n.Status.Conditions
+	for _, condition := range conditions {
+		if condition.Type == v1.NodeReady {
+			node.Properties["status"] = string(condition.Status)
```

Comment:
Here the status will only have True or False. Won't it be better to save it as "Ready"/"NotReady" or "NodeReady"/"NodeNotReady"? Otherwise, simply looking at status (True/False) will not show what it is about.
On the cli, it shows as 'Ready'
<img width="595" alt="image" src="https://github.com/stolostron/search-collector/assets/32376120/ce06e1aa-c4bc-46eb-89ad-211a12813be3">

Reply:
Good catch, I'll change it. The condition can also be "Unknown" and I couldn't find how the CLI displays it, but I'll look again.